styles/header.css

```diff
@@ -0,0 +1,193 @@
+.header {
+  background: url("../assets/heading.png");
+  background-position: center;
+  background-size: cover;
+  padding: 2rem 2rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+.header-cards {
+  flex: 2;
+  height: 60vh;
+  min-height: 520px;
+  position: relative;
+}
```

Comment:
in css rules and conventions we add an empty line between calsses as seperator line

Reply:
add this for all css files